### PR TITLE
Bookmarks page

### PR DESCRIPTION
**Pull request by NilsOt1**
please review :)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -30,10 +30,9 @@ header {
 }
 .nav {
   width: 100%;
-  padding: 0px 0px 30px;
+  padding: 0px 0px 0px;
```
Comment (vaupunkt) [inline css/styles.css:33]: I think, you could delete this line
```suggestion
  
```

File: profil.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    
+    
+    </main>
+    <footer>
+      <nav class="nav">
+        <a class="link" href="index.html"
+          ><img class="home" src="images/house-png-206.png"
+        /></a>
+        <a class="link" href="bookmarks.html"
+          ><img class="bookmark" src="images/bookmark-icon-12326.png"
```
Comment (vaupunkt) [inline profil.html:39]:
```suggestion
          ><img class="bookmark" src="images/bookmark-icon-12323.png"
```

File: bookmarks.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          <p class="tag">#css</p>
+        </section>
+      </section>
+    </main>
+    <footer>
+      <nav class="nav">
+        <a class="link" href="index.html"
+          ><img class="home" src="images/house-png-206.png"
+        /></a>
+        <a class="link active" href="bookmarks"
```
Comment (vaupunkt) [inline bookmarks.html]:
```suggestion
        <a class="link active" href="bookmarks.html"
```

File: bookmarks.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          ><img class="bookmark" src="images/bookmark-icon-12326.png"
```
Comment (vaupunkt) [inline bookmarks.html]: Which Icon did you want to have in your nav-bar? *-12326.png or 12323.png?
```suggestion
          ><img class="bookmark" src="images/bookmark-icon-12326.png"
```

File: css/styles.css (hunk 1 above)
Comment (vaupunkt) [inline css/styles.css]: I think, you could delete this line
```suggestion
 
```

File: profil.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+        <section class="imgname">
+            <figure>
+                <img
+             class="pflanze"
```
Comment (vaupunkt) [inline profil.html:18]: i think "userpicture" or "profileimg" would fit better, because "pflanze" is only a placeholder here

File: profil.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
```
Comment (vaupunkt) [inline profil.html:7]: you could change the title for each site, e.g. "Quiz-App - Userprofile"

File: bookmarks.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
```
Comment (vaupunkt) [inline bookmarks.html:7]: you could change the title for each site, e.g. "Quiz-App - Bookmarks"

File: bookmarks.html [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+      <section class="question">
+        <p class="Frage">What property flips the axes in flexbox?</p>
```
Comment (vaupunkt) [inline bookmarks.html:16]: classes should always start with lowercase-letters
```suggestion
        <p class="frage">What property flips the axes in flexbox?</p>
```
or

```suggestion
        <p class="question">What property flips the axes in flexbox?</p>
```

**Review by vaupunkt — APPROVED** [review]
looks good! you should fix the inconsistent nav-bar icons, but overall a nice design!

File: profil.html (hunk 2 above)
Comment (NilsOt1) [reply profil.html:39]: Gut gesehen :D

File: profil.html (hunk 6 above)
Comment (NilsOt1) [reply profil.html:7]: Gute Idee

**Pull request merged**